src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -334,11 +334,12 @@ protected function loadCacheDriver(string $cacheName, string $objectManagerName,
         if (!isset($cacheDriver['namespace'])) {
             // generate a unique namespace for the given application
             if ($container->hasParameter('cache.prefix.seed')) {
-                $seed = '.'.$container->getParameterBag()->resolveValue($container->getParameter('cache.prefix.seed'));
```

Comment:
the dot should be kept, it's a "namespace" to prevent any collision with auto-generated seeds.

Reply:
I've removed the dot so that the seed stays the same considering the default value in the config `_%kernel.project_dir%.%kernel.container_class%`

Reply:
I think having the default value in the config is important. Not sure what collision you mean. A collision between the default value and someone configuring `prefix_seed` explicitly?